utopia-remix/app/util/copyProjectLink.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { useProjectEditorLink } from './links'
+
+export function useCopyProjectLinkToClipboard() {
+  const projectEditorLink = useProjectEditorLink()
+  return React.useCallback(
+    (projectId: string) => {
+      window.navigator.clipboard.writeText(projectEditorLink(projectId))
+      // TODO notification toast
```

Comment:
TODO for next PR? :)

Reply:
a man can dream 🥹 (seriously I think this needs the whole design - we can chuck them in the operation toasts but that would just be more patchwork)